include/constants/species.h

```diff
@@ -1021,358 +1126,390 @@
+#define SPECIES_SINISTEA_ANTIQUE             FORMS_START + 311
 
 // Polteageist
-#define SPECIES_POLTEAGEIST_ANTIQUE          FORMS_START + 310
+#define SPECIES_POLTEAGEIST_ANTIQUE          FORMS_START + 312
 
 // Alcremie
-#define SPECIES_ALCREMIE_RUBY_CREAM          FORMS_START + 311
-#define SPECIES_ALCREMIE_MATCHA_CREAM        FORMS_START + 312
-#define SPECIES_ALCREMIE_MINT_CREAM          FORMS_START + 313
-#define SPECIES_ALCREMIE_LEMON_CREAM         FORMS_START + 314
-#define SPECIES_ALCREMIE_SALTED_CREAM        FORMS_START + 315
-#define SPECIES_ALCREMIE_RUBY_SWIRL          FORMS_START + 316
-#define SPECIES_ALCREMIE_CARAMEL_SWIRL       FORMS_START + 317
-#define SPECIES_ALCREMIE_RAINBOW_SWIRL       FORMS_START + 318
+#define SPECIES_ALCREMIE_RUBY_CREAM          FORMS_START + 313
+#define SPECIES_ALCREMIE_MATCHA_CREAM        FORMS_START + 314
+#define SPECIES_ALCREMIE_MINT_CREAM          FORMS_START + 315
+#define SPECIES_ALCREMIE_LEMON_CREAM         FORMS_START + 316
+#define SPECIES_ALCREMIE_SALTED_CREAM        FORMS_START + 317
+#define SPECIES_ALCREMIE_RUBY_SWIRL          FORMS_START + 318
+#define SPECIES_ALCREMIE_CARAMEL_SWIRL       FORMS_START + 319
+#define SPECIES_ALCREMIE_RAINBOW_SWIRL       FORMS_START + 320
 
 // Eiscue
-#define SPECIES_EISCUE_NOICE_FACE            FORMS_START + 319
+#define SPECIES_EISCUE_NOICE_FACE            FORMS_START + 321
 
 // Indeedee
-#define SPECIES_INDEEDEE_FEMALE              FORMS_START + 320
+#define SPECIES_INDEEDEE_FEMALE              FORMS_START + 322
 
 // Morpeko
-#define SPECIES_MORPEKO_HANGRY               FORMS_START + 321
+#define SPECIES_MORPEKO_HANGRY               FORMS_START + 323
 
 // Zacian
-#define SPECIES_ZACIAN_CROWNED_SWORD         FORMS_START + 322
+#define SPECIES_ZACIAN_CROWNED_SWORD         FORMS_START + 324
 
 // Zamazenta
-#define SPECIES_ZAMAZENTA_CROWNED_SHIELD     FORMS_START + 323
+#define SPECIES_ZAMAZENTA_CROWNED_SHIELD     FORMS_START + 325
 
 // Eternatus
-#define SPECIES_ETERNATUS_ETERNAMAX          FORMS_START + 324
+#define SPECIES_ETERNATUS_ETERNAMAX          FORMS_START + 326
 
 // Urshifu
-#define SPECIES_URSHIFU_RAPID_STRIKE_STYLE   FORMS_START + 325
+#define SPECIES_URSHIFU_RAPID_STRIKE_STYLE   FORMS_START + 327
 
 // Zarude
-#define SPECIES_ZARUDE_DADA                  FORMS_START + 326
+#define SPECIES_ZARUDE_DADA                  FORMS_START + 328
 
 // Calyrex
-#define SPECIES_CALYREX_ICE_RIDER            FORMS_START + 327
-#define SPECIES_CALYREX_SHADOW_RIDER         FORMS_START + 328
+#define SPECIES_CALYREX_ICE_RIDER            FORMS_START + 329
+#define SPECIES_CALYREX_SHADOW_RIDER         FORMS_START + 330
 
 // Basculegion
-#define SPECIES_BASCULEGION_FEMALE           FORMS_START + 329
+#define SPECIES_BASCULEGION_FEMALE           FORMS_START + 331
+
+// Oinkologne
+#define SPECIES_OINKOLOGNE_FEMALE            FORMS_START + 332
+
+// Maushold
+#define SPECIES_MAUSHOLD_FAMILY_OF_THREE     FORMS_START + 333
```

Comment:
Actually, the family of three form is the base form internally in SV. This may cause other issues if we don't have it the same.

Reply:
Is this the same case for Dudunsparce? The rarer form being the base form, I mean?

Reply:
> Is this the same case for Dudunsparce? The rarer form being the base form, I mean?

Nope, Maushold is the only case.